tests/view_tests/tests/test_csrf.py

```diff
@@ -131,3 +131,7 @@ def test_template_encoding(self):
         with mock.patch.object(Path, "open") as m:
             csrf_failure(mock.MagicMock(), mock.Mock())
             m.assert_called_once_with(encoding="utf-8")
+
+    def test_csrf_response_has_request_context_processor(self):
+        response = self.client.post("/")
+        self.assertIn("request", response.context)
```

Comment:
Thank you for adding a test. I would suggest we make a stronger assertion here, something like:
```suggestion
        self.assertIs(response.wsgi_request, response.context.get("request"))
```

Which ensures both that the `request` key is provided:
```
AssertionError: <WSGIRequest: POST '/'> is not None
```
and that it has the correct value:
```
AssertionError: <WSGIRequest: POST '/'> is not <object object at 0x7f28ced59fe0>
```